obal/data/modules/package_whitelist_check.py

```diff
@@ -0,0 +1,74 @@
+# pylint: disable=C0111,C0301,R1710
+
+try:
+    import configparser
+except:
+    import ConfigParser as configparser
+
+import os
+
+from ansible.module_utils.basic import AnsibleModule  # pylint: disable=C0413
+from ansible.module_utils.obal import get_specfile_name, get_whitelist_status  # pylint:disable=import-error,no-name-in-module
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.2',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+def run_module():
+    module_args = dict(
+        releasers_conf=dict(type='str', required=True),
+        spec_file_path=dict(type='str', required=True),
+        releasers=dict(type='list', required=True),
+        build_command=dict(type='str', required=True),
+    )
+
+    result = dict(
+        changed=False,
+        all_whitelisted=True,
+        branches=[],
+        autobuild_tags=[],
+        whitelist_status=dict(),
+    )
+
+    module = AnsibleModule(
+        argument_spec=module_args,
+        supports_check_mode=True
+    )
+
+    releasers_config = module.params['releasers_conf']
+
+    # fail if we can't find a releasers.conf
+    if not os.path.isfile(releasers_config):
+        module.fail_json(msg="Config file [ {} ] could not be found.".format(releasers_config), **result)
+
+    config = configparser.ConfigParser(allow_no_value=True)
+    config.read(releasers_config)
+
+    package_name = get_specfile_name(module.params['spec_file_path'])
+
+    # get branches and tags
+    for releaser in module.params['releasers']:
+        if config.has_option(releaser, 'branches'): result['branches'] = config.get(releaser, 'branches').split(' ')
```

Comment:
I'd add a new line after `:`. Also note that you're conditionally setting the parameter which means using it as `result['branches']` will raise a `KeyError`.

Reply:
`result['branches']` is set to a default of `[]` above in an attempt to prevent that from occurring.

Reply:
newlines added

Reply:
Right, I missed that :)